tests/kafka/.test.sh

```diff
@@ -0,0 +1,3 @@
+set -e
+
+kafka-topics.sh --list --bootstrap-server localhost:9092
```

Comment:
Where does `kafka-topics.sh` come from?

Reply:
If you prefer we could probably also do a `curl` since the script is probably just calling an endpoint in kafka

Reply:
yeah let's do just curl